nautobot/dcim/models/device_component_templates.py

```diff
@@ -64,8 +64,10 @@ def instantiate(self, device):
         """
         raise NotImplementedError()
 
-    def to_objectchange(self, action):
-        # Annotate the parent DeviceType
+    def to_objectchange(self, action, *, related_object=None, object_data_extra=None, object_data_exclude=None):
+        """
+        Return a new ObjectChange with the `related_object` pinned to the `device_type` by default.
+        """
```

Comment:
Should this be changed to call `super().to_objectchange()` now instead of constructing an `ObjectChange` directly?